packages/optimize/ts/models/Offer.ts

```diff
@@ -81,7 +93,7 @@ class Offer {
     * Note: The Edge sendEvent API can be used to dispatch this data in an Experience Event along with any additional XDM, free-form data, and override
     * dataset identifier.
     * @param {Proposition} proposition - the proposition this Offer belongs to
-    * @return {Promise<Map<string, any>>} - a promise that resolves to xdm map
+    * @return {Promise<Record<string, any>>} - a promise that resolves to xdm map
     */
     generateDisplayInteractionXdm(proposition: Proposition): Promise<Map<string, any>> {        
```

Comment:
You've only updated the API doc to return `Promise<Record<string, any>>`, however the Promise here still resolves to Map<string, any>. Wouldn't this require a conversion?
Same comment for **generateTapInteractionXdm** change below.

Reply:
In React Native the way they were actually coming through was as a object, this was a case where the documentation was incorrect